src/lib389/lib389/_mapped_object.py

```diff
@@ -1301,8 +1301,9 @@ def get(self, selector=[], dn=None, json=False):
             raise ldap.NO_SUCH_OBJECT(f"No object exists given the filter criteria: {criteria} {search_filter}")
         if len(results) > 1:
             entry_dn = [e.dn for e in results]
+            entry_dns_pretty = '\n    '.join(entry_dn);
```

Comment:
Looks like there's an accidental semicolon (`;`) - probably just muscle memory from another language!
It works fine but doesn't match Python's usual style. :)

Reply:
I wonder what other language that could be .... ;)